samples/bluetooth/llpm/README.rst

```diff
@@ -10,6 +10,18 @@ Bluetooth: LLPM
 The Bluetooth® Low Latency Packet Mode (LLPM) sample uses the :ref:`latency_readme` and the :ref:`latency_client_readme` to showcase the LLPM proprietary Bluetooth extension from Nordic Semiconductor.
 You can use it to determine the transmission latency of LLPM-enabled connections, or to compare with different connection parameters and check their influence on the results.
 
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also supports other development kits running SoftDevice Controller variants that support LLPM (see :ref:`nrfxlib:softdevice_controller` Proprietary feature support).
```

Comment:
```suggestion
The sample also supports other development kits running SoftDevice Controller variants that support LLPM (see :ref:`nrfxlib:softdevice_controller` for the proprietary feature support).
```
Not 100% about this, but just "Proprietary feature support" felt a bit odd.

Reply:
I think it was meant to link to the Proprietary feature support table here:  http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8637/nrfxlib/softdevice_controller/README.html#softdevice-controller. I shall try to add a link.

Reply:
Yep, I agree it points to that table. I guess it could work with either the original or my change.

Reply:
Adding a link here: https://github.com/nrfconnect/sdk-nrfxlib/pull/825/files